gui/qt/main_window.py

```diff
@@ -565,11 +568,11 @@ def notify_transactions(self):
                       self.tx_notifications.remove(tx)
                       is_relevant, is_mine, v, fee = self.wallet.get_wallet_delta(tx)
                       if(v > 0):
-                          self.notify(_("New transaction received. %(amount)s") % { 'amount' : self.format_amount_and_units(v)})
+                          self.notify(_("New transaction received: %(amount)s") % { 'amount' : self.format_amount_and_units(v)})
 
     def notify(self, message):
         if self.tray:
-            self.tray.showMessage("Electrum", message, QSystemTrayIcon.Information, 20000)
+            self.tray.showMessage("Electrum", message, QIcon(":icons/electrum_dark_icon"), 20000)
```

Comment:
Have you tested this? Because `QSystemTrayIcon.Information` looks to be an integer.
Anyway, it crashes for me on Debian.

```
TypeError: showMessage(self, str, str, icon: QSystemTrayIcon.MessageIcon = QSystemTrayIcon.Information, msecs: int = 10000): argument 3 has unexpected type 'QIcon'
Aborted
```
[Apparently it's non-trivial to set a custom icon here](https://stackoverflow.com/questions/31045413/qsystemtrayicon-notification-message-with-custom-icon)

Reply:
Oh I see the problem. This requires Qt 5.9.
https://doc.qt.io/qt-5/qsystemtrayicon.html#showMessage-1